api/search.rb

```diff
@@ -23,7 +23,7 @@ def get_thread_ids(context, group_ids, local_params, search_text)
   end
 
   filter.push(
-      {:bool => {:must_not => {:exists => {:field => context}}}},
+      {:bool => {:must_not => {:exists => {:field => :context}}}},
```

Comment:
Is this change correct?  We want the symbol `:context` instead of the variable `context`?

Reply:
Yes, we add condition if field in the ES document is empty